.travis.yml

```diff
@@ -20,8 +20,9 @@ before_script:
   - make setup-dev-env
   - wget "https://dl.minio.io/server/minio/release/linux-amd64/minio"
   - chmod +x minio && nohup ./minio server . &
-  - buffalo db create
-  - buffalo db migrate up
+  - buffalo db create -e $GO_ENV -d -c $POP_PATH/database.yml
```

Comment:
why do we need this if we're removing pop and rdbms storage? 
Same thing for the line below.

Reply:
See below!